applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_isotropic_plasticity_3d.h

```diff
@@ -295,6 +295,23 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) GenericSmallStrainIsotropicPl
     ///@name Protected Operators
     ///@{
 
+    double GetThreshold() { return mThreshold; }

```

Comment:
I am precisely correcting this in one branch, because is not using &

Reply:
But then, if you want to apply these changes in your PR, should I close this one?